test/support/protocol_utils.rb

```diff
@@ -606,6 +604,35 @@ def recv_response
     end
 
     class WebSocketClient
+      class Frame
+        attr_reader :b
```

Comment:
Do you think `bytes` will be  a more expressive name?

Reply:
I'm not sure.
Because `String#b` method exists, I think it's enough, though.

Reply:
So it's intentional to make it feel like `String#b`?

Reply:
Yes. Also, I'm ok with `binary` if you want a longer name. I don't think `bytes` is a good name because`String#bytes` returns an array of bytes.

Reply:
Yeah I think `binary` will be better. That's actually why I asked at the first place: I didn't know what `b` stands for.